.travis.yml

```diff
@@ -31,7 +31,7 @@ after_success:
 
 deploy:
   - provider: script
-    script: make release
+    script: make release images="shipyard-dapper-base"
```

Comment:
we need to pass down image_tag=$TRAVIS_TAG or something like that I think
https://github.com/submariner-io/submariner/blob/master/.travis.yml#L58